codec/src/main/java/io/netty/handler/codec/HeaderDateFormatter.java

```diff
@@ -43,7 +43,7 @@
  * @see <a href="https://tools.ietf.org/html/rfc6265#section-5.1.1">RFC6265</a> for the parsing side
  * @see <a href="https://tools.ietf.org/html/rfc1123#page-55">RFC1123</a> for the encoding side.
  */
-public final class HttpHeaderDateFormatter {
+public final class HeaderDateFormatter {
```

Comment:
If we remove the `Http` portion, then do we need the `Header` portion of the name? Also https://tools.ietf.org/html/rfc6265#section-5.1.1 is specifically for HTTP so should we include this in the name somewhere (e.g. `parseHttpDate`)?